benchmarks/benchmark_experiment.py

```diff
@@ -61,6 +62,12 @@ def list_experiment_configs(self):
         ]
       if self._args.test:
         config_choices["test"] = list(set(self._args.test))
+      if self._args.xla_flags:
+        config_choices["xla_flags"] = list(set(self._args.xla_flags))
+        config_choices["xla_flags"] = [
+            x if x != "None" else None for x in config_choices["xla_flags"]
```

Comment:
nit: you can replace `for x in config_choices["xla_flags"]` with `for x in list(set(config_choices["xla_flags"]))`

Reply:
I assume you mean inlining like so. Did it for the other cases, too. Ty